zomes/chat/src/entries/message.rs

```diff
@@ -64,6 +63,15 @@ pub struct ListMessagesInput {
     pub chunk: Chunk,
     pub active_chatter: bool,
 }
+/// Input to the list messages call
+#[derive(Debug, Clone, Serialize, Deserialize, SerializedBytes)]
+pub struct ListMessagesPageInput {
```

Comment:
```suggestion
pub struct ListMessagesBatchInput {
```
Can we not use the word page?  That makes it seem like we get exact number back.